composer.json

```diff
@@ -37,7 +50,18 @@
       "Tests\\Fixtures\\": "tests/fixtures"
     }
   },
+  "minimum-stability": "dev",
+  "prefer-stable": true,
   "scripts": {
+    "ci": [
```

Comment:
Please drop this script. It's never used.  Same for fix-style. I known what's it for. But we need to keep that simple and same for the rest packagist. 

Instead of `composer run fix-style` use `vendor/bin/codefixer src`.